s1-orbits/README.md

```diff
@@ -1,19 +1,87 @@
+        S1A_OPER_AUX_RESORB_OPOD_20231002T154251_V20231002T115846_20231002T151616.EOF
+        S1A_OPER_AUX_RESORB_OPOD_20231002T171753_V20231002T133730_20231002T165500.EOF
+        ...
+```
+
+### File Naming Convention
+
+The file name convention for these products is `sss_OPER_AUX_tttttt_OPOD_creation_Vstart_stop.EOF`, where:
+- `sss` is the satellite platform (`S1A` or `S1B`)
+- `tttttt` is the orbit type (`POEORB` or `RESORB`)
+- `creation` is the creation date of the file, in CCSDS compact format (`YYYYMMDDTHHMMSS`)
+- `start` is the validity start time of the data contained in the file, in CCSDS compact format (`YYYYMMDDTHHMMSS`)
+- `stop` is the validity stop time of the data contained in the file, in CCSDS compact format (`YYYYMMDDTHHMMSS`)
+
+For example, `S1A_OPER_AUX_POEORB_OPOD_20230821T080724_V20230731T225942_20230802T005942.EOF` is a POEORB POD file for the Sentinel-1A satellite containing data between July 31, 2023 22:59:42 UTC and Aug 2, 2023 00:59:42 UTC. The POD file was created on Aug 21, 2023 at 08:07:24 UTC.
+
+## Tools Supporting Data Access
+
+### AWS S3 Explorer
+
+Browse the contents of the `s1-orbits` bucket in a browser by vising [https://s1-orbits.s3.amazonaws.com/index.html](https://s1-orbits.s3.amazonaws.com/index.html)
+
+### Python Library
+
+[s1_orbits](https://pypi.org/project/s1-orbits/) is a Python library for downloading the best available orbit for a given Sentinel-1 scene.
 
----
+```python
+>>> import s1_orbits
 
-[Registry of Open Data on AWS](https://registry.opendata.aws/)
\ No newline at end of file
+>>> orbit_file = s1_orbits.fetch_for_scene('S1A_IW_SLC__1SDV_20230727T075102_20230727T075131_049606_05F70A_AE0A')
+>>> orbit_file
+PosixPath('S1A_OPER_AUX_POEORB_OPOD_20230816T080815_V20230726T225942_20230728T005942.EOF')
+```
+
+### REST API
+
+For those working outside of Python, https://s1-orbits.asf.alaska.edu/ui provides a REST API for downloading the best available orbit for a given Sentinel-1 scene.
+
+[https://s1-orbits.asf.alaska.edu/scene/S1A_IW_SLC__1SDV_20230727T075102_20230727T075131_049606_05F70A_AE0A](https://s1-orbits.asf.alaska.edu/scene/S1A_IW_SLC__1SDV_20230727T075102_20230727T075131_049606_05F70A_AE0A)
+
+### SNS Subscriptions
+
+The SNS topic `arn:aws:sns:us-west-2:211125554030:s1-orbits-object_created` provides push notifications when new files are added to the `s1-orbits` S3 bucket. The topic supports SQS and Lambda subscriptions.
+
+- To subscribe a Lambda Function, see [Invoking Lambda functions with Amazon SNS notifications](https://docs.aws.amazon.com/lambda/latest/dg/with-sns.html).
+- To subscribe an SQS Queue, see [Subscribing a queue to an Amazon SNS topic using the Amazon SQS console](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-configure-subscribe-queue-sns-topic.html).
+
+SNS messages conform to the S3 [event message structure](https://docs.aws.amazon.com/AmazonS3/latest/userguide/notification-content-structure.html).
```

Comment:
Here's an example message, but this feels slightly too verbose to include in the readme.

```json
{
  "Records": [
    {
      "eventVersion": "2.1",
      "eventSource": "aws:s3",
      "awsRegion": "us-west-2",
      "eventTime": "2024-08-09T19:12:57.513Z",
      "eventName": "ObjectCreated:Put",
      "userIdentity": {
        "principalId": "AWS:AROA4JSBULMN3PFAYTH26:s1-orbits-Fetcher-1XJD1T70P3N5H-Lambda-1xLx9Lz2n279"
      },
      "requestParameters": {
        "sourceIPAddress": "35.81.77.244"
      },
      "responseElements": {
        "x-amz-request-id": "8A9QZC3YM3CZYKZ1",
        "x-amz-id-2": "r5fb1iOv2KmQMBdEWGSyn49zKjhkIt+YYfHGx0FJTproOIVWpVm+9e90vR/fXtuVSC4EbFq0xvG4lu/hMXtA9vegNuU67p5a"
      },
      "s3": {
        "s3SchemaVersion": "1.0",
        "configurationId": "ZmZiNmEyODAtMzM4YS00YmE4LTgzZWYtYmRkYWVkYWIxMTZl",
        "bucket": {
          "name": "s1-orbits",
          "ownerIdentity": {
            "principalId": "A382N192PC5B9F"
          },
          "arn": "arn:aws:s3:::s1-orbits"
        },
        "object": {
          "key": "AUX_RESORB/S1A_OPER_AUX_RESORB_OPOD_20240809T190345_V20240809T151609_20240809T183339.EOF",
          "size": 590469,
          "eTag": "bf4ddadc91d85f4404866247e44fe923",
          "sequencer": "0066B66A3954EB3E5D"
        }
      }
    }
  ]
}
```

Reply:
Yeah I agree, a bit too verbose.